JustSaying/CreateMeABus.cs

```diff
@@ -9,10 +9,12 @@ public static class CreateMeABus
         /// <summary>
         /// Allows to override default <see cref="IAwsClientFactory"/> globally.
         /// </summary>
+        ////[Obsolete("Use the BusBuilder class to create message buses.")]
         public static Func<IAwsClientFactory> DefaultClientFactory { get; set; }
             = () => new DefaultAwsClientFactory();
 
-        public static JustSayingFluentlyDependencies WithLogging(ILoggerFactory loggerFactory) => 
+        ////[Obsolete("Use the BusBuilder class to create message buses.")]
```

Comment:
Is `////` intentional?

Reply:
Habit to keep StyleCop happy, but I should revert this change as it’s not so much obsolete as gonna be deleted later I guess?